examples/core/src/main/scala/com/twitter/util/LastWriteWinsQueue.scala

```diff
@@ -41,30 +41,30 @@ class LastWriteWinsQueue[A] extends java.util.Queue[A] {
     } else Array[Any]().asInstanceOf[Array[T with java.lang.Object]]
   }
 
-  def toArray = toArray(new Array[AnyRef](0))
+  def toArray: Array[AnyRef with Object] = toArray(new Array[AnyRef](0))
```

Comment:
Oof, this `AnyRef with X` thing looks pretty awkward.

Reply:
This one specifically is a bit different because that is the user-defined return type of `toArray`, though we can shorten this as well